.github/set-min-requirements.py

```diff
@@ -2,8 +2,8 @@
     "requirements.txt",
     "requirements/extra.txt",
     "requirements/loggers.txt",
-    "requirements/test.txt",
     "requirements/examples.txt",
+    # "requirements/test.txt",  # Don't use old testing packages
```

Comment:
Just commenting it out in case someone thinks it's missing, but could also be deleted completely.